api/Node.json

```diff
@@ -135,10 +135,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "7"
+              "version_added": "≤4"
```

Comment:
I can't figure out from source when this was added, so leaving as a range seems fine.